configs/default/LMNR-LUXMINIF7.config

```diff
@@ -1,8 +1,8 @@
-# Betaflight / STM32F7X2 (S7X2) 4.1.0 Sep 25 2019 / 14:09:57 (08e8afa09) MSP API: 1.42
-# manufacturer_id: AIRB   board_name: LUXMINIF7   custom defaults: NO
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Oct  6 2019 / 06:48:40 (c6452a55c) MSP API: 1.42
+# manufacturer_id: LMNR   board_name: LUXMINIF7   custom defaults: NO
 
 board_name LUXMINIF7
-manufacturer_id AIRB
+manufacturer_id LMNR
```

Comment:
@x4FF3: Should we put you down as the target maintainer for Lumenier as well?

Reply:
yes please.